### DISCUSSION TITLE
antd源码在本地编译报错

### DISCUSSION THREAD

**Discussion by ChenKun1997** [ant-design/ant-design#34204] [Q&A] [answered]
> antd版本：4.18.8  
> 系统：macOS Big Sur 11.6  
> node版本：v16..14.0

在本地运行npm install之后，npm start,报错如下
![image](https://user-images.githubusercontent.com/24249578/155641894-d0371978-96bc-43be-9657-1645000bc699.png)
在谷歌斌还没有找到其他解决办法

**Comment by yoyo837** [comment]
node和npm版本不兼容吧？啥版本

**Reply by ChenKun1997** [reply]
![image](https://user-images.githubusercontent.com/24249578/155642606-30a37524-16ca-44eb-be6b-4a7e49667c62.png)

我这个是兼容的，我查看了node官网，这个版本的node对应npm版本

![image](https://user-images.githubusercontent.com/24249578/155644565-03f02756-c333-4592-8c0f-121acb858c6b.png)

**Comment by yoyo837** [comment]
我的也不行了。😅

**Reply by ChenKun1997** [reply]
现在有什么方法可以解决呢，或者是去哪里提问会有大佬来解决 。:sweat_smile:

**Reply by yoyo837** [reply]
我去群里求助一下看看。

**Reply by ChenKun1997** [reply]
@yoyo837 我知道什么问题了，我之前用的时node版本时16，换成14就没问题了，官方文档也没说明，我是在stackoverflow求助的

**Reply by Dashuai-Black** [reply]
> 我去群里求助一下看看。

老哥 是antd的群么 可以拉下么

**Comment by ChenKun1997** [comment, marked as answer]
node版本换成14就没问题了
